tutorials/preprocessing/plot_55_setting_eeg_reference.py

```diff
@@ -0,0 +1,196 @@
+   :depth: 2
+
+As usual we'll start by importing the modules we need, loading some
+:ref:`example data <sample-dataset>`, and cropping it to save memory. Since
+this tutorial deals specifically with EEG, we'll also restrict the dataset to
+just a few EEG channels so the plots are easier to see (using
+:meth:`~mne.io.Raw.pick`):
+"""
+
+import os
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+raw.crop(tmax=60).load_data()
+raw.pick(['EEG 0{:02}'.format(n) for n in range(41, 60)])
+
+###############################################################################
+# Background
+# ^^^^^^^^^^
+#
+# EEG measures a voltage (difference in electric potential) between each
+# electrode and a reference electrode. This means that whatever signal is
+# present at the reference electrode is effectively subtracted from all the
+# measurement electrodes. Therefore, an ideal reference signal is one that
+# captures *none* of the brain-specific fluctuations in electric potential,
+# while capturing *all* of the environmental noise/interference that is being
+# picked up by the measurement electrodes.
+#
+# In practice, this means that the reference electrode is often placed in a
+# location on the subject's body and close to their head (so that any
+# environmental interference affects the reference and measurement electrodes
+# similarly) but as far away from the neural sources as possible (so that the
+# reference signal doesn't pick up brain-based fluctuations). Typical reference
+# locations are the subject's earlobe, nose, mastoid process, or collarbone.
+#
+# Even in cases where no electrode is specifically designated as the reference,
+# EEG recording hardware will still treat one of the scalp electrodes as the
+# reference, and the recording software may or may not display it to you (it
+# might appear as a completely flat channel, or the software might subtract out
+# the average of all signals before displaying, making it *look like* there is
+# no reference).
+#
+#
+# Setting or changing the reference channel
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# If you want to recompute your data with a different reference than was used
+# when the raw data were recorded / saved, MNE-Python provides the
+# :meth:`~mne.io.Raw.set_eeg_reference` method on :class:`~mne.io.Raw` objects,
+# as well as the :func:`mne.add_reference_channels` function. To use an
+# existing channel as the new reference, use the
+# :meth:`~mne.io.Raw.set_eeg_reference` method; you can also designate multiple
+# existing electrodes as reference channels, as is sometimes done with mastoid
+# references:
+
+# code lines below are commented out because the sample data doesn't have
+# earlobe or mastoid channels, so this is just for demonstration purposes:
```

Comment:
Some analyses use Cz as reference. Might use something like that here.

Reply:
Yeah...  I wanted to use `Cz` in the following paragraph; that's where I think it makes the most sense. I really wish our sample data used 10-20 or 10-05 channel names.